### ISSUE TITLE
List of teams in fac dashb not same as teams list in chat

### ISSUE THREAD

**Issue by branscombj** [lzim/teampsd#2017]
**1. Add description: List of "my teams" in the facilitator dashboard vs. list of "my teams" inside the Team Chat when I'm in the St. George Team world** 

![image](https://user-images.githubusercontent.com/29778290/119535983-946de800-bd56-11eb-975d-312cecc6b72a.png)
![image](https://user-images.githubusercontent.com/29778290/119536040-a3ed3100-bd56-11eb-8b89-2016bfbf2242.png)
![image](https://user-images.githubusercontent.com/29778290/119536609-44435580-bd57-11eb-85d7-c8528e9af1be.png)

 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
iix. **Releases** - assign to key team release grouped with other relevant issues  

**3. If you anticipate that your **issue card** or **bug card** will have **dependencies** these can be added once the issue has been submitted.   
ix. **Add dependencies** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by jamesmrollins** [comment]
Hi @branscombj .  After more careful review of the issue, I want to confirm I have the right mental model.  My understanding of your observation is that your listing does not contain the name_teamname_ind (i.e., james_640a0_pct_ind).  In other words, the listing doesn't contain the individual's world, in the _ind group.  

**If this is true, then please read on.** You can chat with any end-user registered in Epicenter, by clicking the chat icon and entering their user name (their VA email, in this case) in the "Enter Username" field (see below).  This will open a private session with that person.  The reason they are not listed in the teams, is because teams is for maintaining threads with everyone in the group - it is not private; rather, made for addressing multiple people at once. The individual worlds (name_teamname_ind) are not teams - so they don't appear in the listing.

Does that make sense?

### How to set up a private chat:
![image](https://user-images.githubusercontent.com/30132017/119570475-03ead400-bd65-11eb-90ef-bba9b242df0b.png)



FYI @lijenn @staceypark

**Comment by branscombj** [comment]
Hi @jamesmrollins - That is great info to know! But no, I don't think it's quite on the mark as far as my question.
In the screenshots above, first there are 2 that (together) show all the teams I can choose from my fac dashboard. That looks correct; it's got St. George's team world and all the St. George team members' ind worlds; it has KC team (ind worlds are probably not set up yet as we haven't gotten that far in MTL). Then it has my own ind world, and another stacey/jane/ind one that's probably from @staceypark doing some testing...

Then in the 3rd screenshot you see what options I have in the forio chat - at least when I selected St. George Team world to log into from the fac dash. My questions are about that list:
1. Why so different from the above list? Is it missing all the other St. George inds because those "teams" haven't chatted anything yet?
2. Why include several teams from old work - Team2, Team4, Facilitate_Pilot, etc. Maybe it should retain access to all past chats for research purposes etc. - so that might make sense. Although if so, the list could get long fast.
3. When I highlight one team - in this case, 660gg_st_george_cboc_ind - I can see and participate in the chat in that world - in this case it looks like it's Al Hernandez's ind world, although I wouldn't be able to distinguish it from the team name. So it looks like Al was in his ind world and he entered a chat - even though it's his ind world so he probably was just testing it out, thinking he was speaking to himself. But as I said in my comment to him, i did see a red "badge" indicating there was a chat to read, in the top menu as soon as I entered the St. George team world.

I am not sure there is a bug here; this may be more about "how does this work"!

**Comment by jamesmrollins** [comment]
Hi @branscombj - Answers to your questions are below:

_1.  Why so different from the above list? Is it missing all the other St. George inds because those "teams" haven't chatted anything yet?_

**Because the username_ind are not teams and shouldn't be on the teams listing in the chat. teamname_ind are teams and you can address the whole team using the team section in chat. Username_ind are listed on your facilitator dashboard because you can visit their individual session to help them.**

_2. Why include several teams from old work - Team2, Team4, Facilitate_Pilot, etc. Maybe it should retain access to all past chats for research purposes etc. - so that might make sense. Although if so, the list could get long fast._

**I agree the list will become quickly unmanageable if we don't implement some housekeeping. @lijenn and I should remove all the esoteric groups from TEST and PROD to streamline the list. The software uses your assignment and whatever is in Epicenter to build these lists.**

_3. When I highlight one team - in this case, 660gg_st_george_cboc_ind - I can see and participate in the chat in that world - in this case it looks like it's Al Hernandez's ind world, although I wouldn't be able to distinguish it from the team name. So it looks like Al was in his ind world and he entered a chat - even though it's his ind world so he probably was just testing it out, thinking he was speaking to himself. But as I said in my comment to him, i did see a red "badge" indicating there was a chat to read, in the top menu as soon as I entered the St. George team world._

**This is the correct operation.  When Al Hernandez entered a chat in the 660gg_st_george_cboc_ind team, everyone in that team received his chat including you.  When you responded, everyone in that team including Al got your response.  Again, the use of the teams discussion threads are like an intercom for addressing everyone in the team (just like Microsoft Teams).**

Does this help clear things up?

**Comment by branscombj** [comment]
@jamesmrollins So, all the IND worlds of the St George team members are kind of within one uber-world? That's why in pix 1 & 2 they're indented under 660gg_..._team, and why in pic 3 it doesn't differentiate that it's Al Hernandez's 660gg_..._ind - is that right?

**Comment by jamesmrollins** [comment]
Your knowledge of the Sim UI is never a low priority for me :-) **You are correct.**  Uber-worlds are called "groups" in Epicenter, and "Teams" in _mtl_. Typically, teams (aka-groups) can have multiple endusers in one world, or multiple endusers in several worlds.

**Comment by jamesmrollins** [comment]
The teams procedures discussed above are now documented in Chapter 8 of the mtl.manual.
